CheckDrive.Mobile/CheckDrive.Mobile/CheckDrive.Mobile/ViewModels/ChatViewModel.cs

```diff
@@ -0,0 +1,43 @@
+﻿using CheckDrive.Mobile.Models;
+using System;
+using System.Collections.ObjectModel;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace CheckDrive.Mobile.ViewModels
```

Comment:
here too  ;  It would be good to write down

Reply:
Xamarin forms doesn't support this technology